requirements/edx/base.txt

```diff
@@ -476,7 +476,7 @@ edx-rest-api-client==5.4.0
     #   edx-proctoring
 edx-search==3.1.0
     # via -r requirements/edx/base.in
-edx-sga==0.16.0
+edx-sga==0.17.2
```

Comment:
[Sandbox](https://tools-edx-jenkins.edx.org/job/sandboxes/job/CreateSandbox/42490/) in progress to test the new version before upgrading.

Reply:
Tested `staff graded submission` problem submission and grading to be working correctly on the sandbox.